site/src/pages/docs/server-flags-provider.mdx

```diff
@@ -0,0 +1,66 @@
+# Configuring system properties
+
+You can configure the system properties by implementing your own <type://FlagsProvider>. It is useful for following two situations:
```

Comment:
- How about starting the section with 'what flags in Armeria are'? 
- We could also mention that a user can override these properties by setting system properties (with a few examples).